### DISCUSSION TITLE
Civil Engineering Open Source

### DISCUSSION THREAD

**Discussion by CivilEngEC** [community/community#50734] [Programming Help]
### Select Topic Area

Show & Tell

### Body

Is anyone familiar with a free, open-source program for budgeting projects in python or another language? I will like to contribute. I have been looking, but I haven´t found any. So far, I have found a few people working on free open-source for civil engineering, so I have started by creating my repository as a hobby. I invite anyone interested to look into it and contribute, or if someone is already working on it, I would like to work with them in my free time.

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @CivilEngEC ! 

We’ve moved your post to our new Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.

**Reply by CivilEngEC** [reply]
Thank you @queenofcorgis
